integrations/virtual_camera/__init__.py

```
@@ -1,57 +1,51 @@
->>> cam = VirtualCamera(device="/dev/video0", fps=30, resolution=(1280, 720))
->>> cam.push_frame(composited_pil_image)
->>> cam.start()
->>> cam.stop()
-
-Implementation notes
---------------------
-- Linux: uses v4l2loopback and pyfakewebcam.
-- macOS: requires the OBS DAL plugin or a third-party virtual camera.
-- Windows: requires OBS Virtual Camera or equivalent driver.
-- This module should NOT depend on the bot layer.
+integrations/virtual_camera/__init__.py – virtual camera state adapter.
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)